doc/release-process.md

```diff
@@ -2,6 +2,19 @@ Release Process
 ====================
 Meta: There should always be a single release engineer to disambiguate responsibility.
 
+## Pre-release
+
+Check all of the following:
+
+- All dependencies have been updated as appropriate:
+  - BDB
+  - Boost
+  - ccache
+  - libgmp
+  - libsodium
+  - miniupnpc
+  - OpenSSL
+
 ## A. Define the release version as:
 
     $ ZCASH_RELEASE=${UPSTREAM_VERSION}.z${ZCASH_RELEASE_COUNTER}
```

Comment:
@str4d could you update the ci.leastauthority.com URL in `doc/release-process.md` to https://ci.z.cash?